test_runner/src/main/kotlin/ftl/cli/firebase/test/ios/IosRunCommand.kt

```diff
@@ -39,23 +39,15 @@ class IosRunCommand : CommonRunCommand(), Runnable {
         val config = IosArgs.load(Paths.get(configPath), cli = this)
 
         if (dumpShards) {
-            val testShardChunksJson: String = prettyPrint.toJson(config.testShardChunks)
-            Files.write(Paths.get(shardFile), testShardChunksJson.toByteArray())
-            println("Saved shards to $shardFile")
-        } else {
-            runBlocking {
-                newTestRun(config)
-            }
+            dumpShards(config)
+        } else runBlocking {
+            newTestRun(config)
         }
     }
 
-    companion object {
-        private const val shardFile = "ios_shards.json"
-    }
-
     // Flank debug
 
-    @Option(names = ["--dump-shards"], description = ["Dumps the shards to $shardFile for debugging"])
+    @Option(names = ["--dump-shards"], description = ["Dumps the shards to $ANDROID_SHARD_FILE for debugging"])
```

Comment:
`IosRunCommand.kt` references `ANDROID_SHARD_FILE`, is that correct?

Reply:
shame on me...